distributed/scheduler.py

```diff
@@ -385,12 +387,81 @@ def mark_ready_to_run(self, key):
                 self.unrunnable.add(key)
             else:
                 self.stacks[new_worker].append(key)
-                self.maybe_ready.add(new_worker)
+                self.maybe_idle.add(new_worker)
                 # self.ensure_occupied(new_worker)
         else:
             self.ready.appendleft(key)
             # self.ensure_idle_ready()
 
+    def should_steal(self, key, bandwidth=100e6):
```

Comment:
Added a module level constant.  I think we should specify these sorts of things with config files eventually when there are other such parameters like choice of compression.  

At the moment I'm not too concerned with bandwidth for work-stealing.  For most tasks I've seen the computation / communication ratio is either very high or very low.